edit.html.haml

```diff
@@ -0,0 +1,138 @@
+.content
+  = render :partial => 'users/nav_bar', :locals => { :current_user => current_user }
+
+.content
+  = render :partial => 'users/flash_messages', :locals => { :flash => flash }
+
+.container.title-container
+  %h2 Update Your Account
+
+
+-# .container.form-container
+
+= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put, multipart: true }) do |f|
+  = render "devise/shared/error_messages", resource: resource
+
+  .container.form-container
+    .row
+      .col-md-4.form-img-wrapper
+        = image_tag @user.image.url
+
+        %label.cabinet.btn.btn-primary
+          .text
+            Upload new image
+          = f.file_field :image, class: %w[file]
+
+
+        -# .img-upload.btn.btn-primary
```

Comment:
Is this for future use/extension? If not, I can delete it as part of maintenance.

Reply:
this file was a mistake. Accidentally hit paste while going through folders, didn't find what i pasted